react/src/App.tsx

```diff
@@ -64,7 +65,7 @@ const useLoadAndSetAvailableBases = () => {
       const organisationId = decodedToken["https://www.boxtribute.com/organisation_id"];
       dispatch({
         type: "setOrganisationId",
-        payload: organisationId,
+        payload: organisationId ?? 1,
```

Comment:
I assume this is for the god user who does not have an organisation ID set. Is it possible to use 0 as a value? 1 is ambiguous (even though it's a test org).

Reply:
@pylipp actually it ensures that only available organization are selected for god user. If it is set to 0, then the source organisation name and base information cannot be shown.